app/transformers/api/current/appointment_transformer.rb

```diff
@@ -1,11 +1,2 @@
 class Api::Current::AppointmentTransformer
```

Comment:
This class should inherit the generic transformer, and the current controller should use this instead.

Reply:
Fixed.